### PR TITLE
Update monero to 0.12.0.0

### PR DESCRIPTION
**Pull request by sammy007**
There is a HF April 6.

### CODE REVIEW

**Comment by sammy007** [comment]
https://github.com/Homebrew/homebrew-core/pull/4052

**Comment by fxcoudert** [comment]
The build is now OK, but the test fails with:

```
==> /usr/local/Cellar/monero/0.12.0.0/bin/monero-wallet-cli --restore-deterministic-wallet --password brew-test --restore-height 1 --generate-new-wallet wallet --electrum-seed 'baptism cousin whole exquisite bobsled fuselage left scoop emerge puzzled diet reinvest basin feast nautical upon mullet ponies sixteen refer enhanced maul aztec bemused basin'--command address
Error: monero: failed
<0> expected but was
<1>.
```

**Comment by fxcoudert** [comment]
Is it normal that it gives a 1 exit code (where previously it had 0)?

**Comment by sammy007** [comment]
I got 0 exit code in a shell.

File: Formula/monero.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Monero < Formula
-  desc "Official monero wallet and cpu miner"
+  desc "Secure, private, untraceable cryptocurrency"
   homepage "https://getmonero.org/"
-  url "https://github.com/monero-project/monero/archive/v0.11.1.0.tar.gz"
-  sha256 "b5b48d3e5317c599e1499278580e9a6ba3afc3536f4064fcf7b20840066a509b"
+  url "https://github.com/monero-project/monero.git",
+      :tag => "v0.12.0.0",
```
Comment (woodruffw) [inline Formula/monero.rb]: Please use the [tarball URL](https://github.com/monero-project/monero/archive/v0.12.0.0.tar.gz) here, and not a `git` tag/commit.

**Review by woodruffw — CHANGES_REQUESTED** [review]

File: Formula/monero.rb (hunk 1 above)
Comment (sammy007) [reply Formula/monero.rb]: There is git submodule and tarball is missing it.

**Pull request closed without merging**